holoviews/plotting/bokeh/element.py

```diff
@@ -746,7 +746,7 @@ def initialize_plot(self, ranges=None, plot=None, plots=None, source=None):
             element = [el for el in self.hmap.data.values() if el][-1]
         else:
             element = self.hmap.last
-        key = self.keys[-1]
+        key = util.wrap_tuple(self.hmap.last_key)
```

Comment:
Trying to understand the different between these two...

Reply:
The former is a global key across a whole layout, the latter the last actually existing key on the subplot. The former would have been causing errors in some cases since you can't initialize on a non-existent key.

Reply:
Thanks for clarifying...